cms/src/test/java/uk/nhs/digital/externalstorage/workflow/externalFileUnpublish/ExternalFileUnpublishTaskTest.java

```diff
@@ -42,22 +54,56 @@ public void setUp() throws RepositoryException {
         Repository repository = MockJcr.newRepository();
         session = repository.login();
         rootNode = session.getRootNode();
+        testFolder = rootNode.addNode("lorem-ipsum", NT_FOLDER);
         externalFileUnpublishTask = new ExternalFileUnpublishTask();
-        externalFileUnpublishTask.setS3Connector(s3Connector);
 
         given(workflowContext.getInternalWorkflowSession()).willReturn(session);
         externalFileUnpublishTask.setWorkflowContext(workflowContext);
     }
 
     @Test
-    public void shouldCallS3() throws WorkflowException, RepositoryException {
-        Node docNode = rootNode.addNode("test-document");
-        addVariantNode(docNode, PUBLISHED);
-        MockJcr.setQueryResult(session, getQueryResults());
+    public void shouldUnpublishResources() throws WorkflowException, RepositoryException {
```

Comment:
Isn't this test made redundant by `shouldUnpublishPublicationAndItsDatasetsResources`?

Unless the intention was to emphasise that it's about testing a 'standalone' publication, in which case the method's name could use changing to indicate this.

Reply:
renaming to `shouldUnpublishAnyDocumentResources`